roles/services/tasks/main.yml

```diff
@@ -10,6 +10,19 @@
   with_items:
     - postfix
 
+- block:
+  - name: ensure file /etc/containers/containers.conf exists
+    file:
+      path: /etc/containers/containers.conf
+      state: touch
+      mode: 0644
+
+  - name: Add default capabilities
+    lineinfile:
+      path: /etc/containers/containers.conf
+      line: "default_capabilities = [\n\t\"SYS_CHROOT\",\n]"
```

Comment:
Multiple issues with this:

1. You are missing the section header (`[containers]`) and the value will be ignored. You should be getting warnings about this.
2. With the section added, only the `SYS_CHROOT` capability will be added. The rest (if there are some defined elsewhere), will not be used at all.

Reply:
I am not sure what capabilites are there by default on the TF machines, but on my local host there is:
```
CHOWN,DAC_OVERRIDE,FOWNER,FSETID,KILL,NET_BIND_SERVICE,SETFCAP,SETGID,SETPCAP,SETUID
```

Reply:
Note that both of these issues are based on my podman version, so there might be a change of behavior with how the config files are handled in newer versions. Feel free to correct me if that is the case.

Reply:
Hmm, based on the comment from `containers.conf` file. `ontainer engines will read containers.conf files in up to three
# locations in the following order:
#  1. /usr/share/containers/containers.conf
#  2. /etc/containers/containers.conf
#  3. $HOME/.config/containers/containers.conf (Rootless containers ONLY)` It looks like we need to either modify /usr/share/containers/containers.conf file or copy whole section [containers] to /etc/containers/containers.conf file.

Reply:
It looks like we have to look at file /usr/share/containers/containers.conf file and check if row `default_capabilities = [` exists and replace it with `default_capabilities = [\n\t\"SYS_CHROOT\"`

Reply:
Fixed by https://github.com/sclorg/sclorg-testing-farm/pull/61/commits/2845c8cbb11afab2eddd4e42538226a8ab3081a2. Let's modify /usr/share/containers/containers.conf file.